samples/Log4net/BasicAWSCredentialsConfigurationExample/BasicAWSCredentialsConfigurationExample.csproj

```diff
@@ -32,7 +32,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="AWSSDK.Core, Version=3.3.0.0, Culture=neutral, PublicKeyToken=885c28607f98e604, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\AWSSDK.Core.3.3.27\lib\net45\AWSSDK.Core.dll</HintPath>
+      <HintPath>..\..\..\packages\AWSSDK.Core.3.5.1.22\lib\net45\AWSSDK.Core.dll</HintPath>
```

Comment:
Smell test:  Did you mean to update the `<Reference>` parent node when you updated the `<HintPath>`?    I would expect the version of `AWSSDK.Core` to be `3.5.x.x`